### DISCUSSION TITLE
Any reductions in housemate transmission or infection?

### DISCUSSION THREAD

**Discussion by kdbscott** [microCOVID/microCOVID#967] [Q&A] [answered]
I think the spreadsheet currently assumes that if one housemate has COVID (Delta), there is a 40% chance that the other housemate will get COVID.

But if both housemates are vaccinated, wouldn't the chances be lower? Both from reduction in transmissibility and infection rate? 

After briefly looking into this, it seems like initial data are uncertain about changes in transmissibility (https://www.nature.com/articles/d41586-021-02187-1), but I'd be willing to bet that it does to some extent. But even if we assume it doesn't and I still give my housemate N covid particles, wouldn't it still be less likely they catch covid given N particles?

**Comment by blanchardjeremy** [comment, marked as answer]
Good question. Let's break it down.

- Reduction in transmission: Our current modeling doesn't suggest any reduction in transmission after being vaccinated. The next two points do apply, though, to reduce the overall risk situation.
- Reduction in exposure: You automatically are exposed to fewer microCOVIDs worth of risk because the people in your house are accumulating less. This takes shape most notably by way of the next bullet:
- Change in "housemate budgets": The spreadsheet doesn't yet have a feature for this, but you can effectively change "housemate budget reductions" formula based on everyone being vaccinated. You can see the formula here: https://www.microcovid.org/paper/13-q-and-a#2-if-a-vaccinated-individual-contracts-covid-how-much-less-or-more-likely-is-this-to-result-in-negative-consequences-increased-budget

Hope this helps
